sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -52,6 +53,15 @@ import org.apache.spark.util.collection.BitSet
 /** Used by [[TreeNode.getNodeNumbered]] when traversing the tree for a given number */
 private class MutableInt(var i: Int)
 
+/** The class represents error context of a SQL query. */
+case class SqlQueryContext(
+    override val objectType: String,
+    override val objectName: String,
+    override val startIndex: Int,
+    override val stopIndex: Int,
+    override val fragment: String,
+    override val summary: String) extends QueryContext
```

Comment:
I think `QueryContext` should not have `summary`

Reply:
Yep, removed it.